composer.json

```diff
@@ -18,6 +18,8 @@
     "drupal/address": "^1.10",
     "drupal/core-composer-scaffold": "^9.2",
     "drupal/core-recommended": "^9.2",
+    "drupal/migrate_plus": "^6.0",
```

Comment:
We need to keep these in a different composer.json file within the yalesites_profile. This system of organization is still evolving, but for now anything drupaly belongs in the profile while infrastructure and core tools go into the top level composer file.